dapps/src/handlers/fetch.rs

```diff
@@ -303,7 +307,7 @@ impl ContentFetcherHandler {
 					Ok(ValidatorResponse::Streaming(stream)) => {
 						trace!(target: "dapps", "Validation OK. Streaming response.");
 						let (reading, response) = stream.into_response();
-						fetch2.process_and_forget(reading);
+						pool.spawn(reading).forget();
```

Comment:
That could probably be pass-through (Since we use `BodyParser` to convert async->sync and later we use `into_response()` to convert from sync->async). To be investigated in subsequent PRs.